projects/valtimo/components/src/public_api.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015-2020 Ritense BV, the Netherlands.
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
```

Comment:
2022 is fine but unnecessary, it's still being looked at if we should update copyright notices